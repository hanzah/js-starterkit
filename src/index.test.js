describe('Test sum', () => {
	it('1+1', () => {
		expect(1+1).toBe(2);
	})

	it('1+2', () => {
		expect(1+2).toMatchSnapshot();
	})
})
