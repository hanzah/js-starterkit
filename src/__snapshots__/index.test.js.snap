exports[`Test sum 1+2 1`] = `3`;
